codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/sra/test-aws-json-async-client-class.java

```diff
@@ -128,7 +128,7 @@ final class DefaultJsonAsyncClient implements JsonAsyncClient {
     private static final Logger log = LoggerFactory.getLogger(DefaultJsonAsyncClient.class);
 
     private static final AwsProtocolMetadata protocolMetadata = AwsProtocolMetadata.builder()
-            .serviceProtocol(AwsServiceProtocol.AWS_JSON).build();
+                                                                                   .serviceProtocol(AwsServiceProtocol.AWS_JSON).build();
```

Comment:
In the future, can we avoid whitespace formatting changes for codegen files? Hard to see the actual changes with them. Copy and "Paste as Plain Text" should do the trick

Reply:
Yeah, I was originally use a small script to update these automatically (should be the same formatting as paste as plain text), but was seeing a lot of small formatting changes.  Copy/paste seemed to get it closer.... but I think it depends on how the last person updated the file.